calibration/ADCPAM/CGINS-ADCPAM-73246__20210304.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+673246,CC_scale_factor1,0.61,
+673246,CC_scale_factor2,0.61,
+673246,CC_scale_factor3,0.61,
+673246,CC_scale_factor4,0.61,

```

Comment:
looks correct, but no calibration/configuration file available on vault to verify date.